DCA/run.py

```diff
@@ -0,0 +1,30 @@
+from skimage.io import imread
+import numpy as np
+
+from pathlib import Path
+import os
+
+from adapter_utils import max_power_of_2, get_args
+from adapter import DcaAdapter
+
+
+if __name__ == "__main__":
+    print("Starting work")
+    args = get_args()
+
+    if args.device.lower() == 'cuda' or args.device.lower() == 'gpu':
+        os.environ["CUDA_VISIBLE_DEVICES"] = "0"
```

Comment:
It's not a good idea to change the environment from Python code. This variable must be set by the user before starting if he wants to work with the GPU